.github/workflows/rust.yml

```
@@ -0,0 +1,15 @@
+name: Rust
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build
+      run: cargo build --verbose
+    - name: Run tests
+      run: cargo test --verbose
```

I'm not convinced that `--verbose` is helpful in most cases. How often do we need to look through _successful_ tests. It's much more usual that we want to see any failing tests in a CI run.
```suggestion
      run: cargo test
```

---

Yeah, I think I would prefer turning off Verbose and turning on BACKTRACE (can't remember the exact name of that var).